src/helics/application_api/MessageFederate.cpp

```diff
@@ -313,11 +313,6 @@ void MessageFederate::setMessageNotificationCallback (const Endpoint &ept,
 /** get a count of the number endpoints registered*/
 int MessageFederate::getEndpointCount () const { return mfManager->getEndpointCount (); }
 
-void MessageFederate::setEndpointOption (const Endpoint &ept, int32_t option, bool option_value)
```

Comment:
This is the only change I see in MessageFederate.cpp. What I don't see is how the flags are set for an Endpoint when in MessageFederate::registerInterfacesJson/Toml(const std::string &jsonString) it isn't checking for a "flags" key in that function nor calling a function similar to loadOptions found in federate.cpp.

Reply:
I think I skipped over that since the only valid flags on endpoints are required/optional  but I will add in the logic for checking the flags.

Reply:
there should be a loadOptions function in MessageFederate.cpp now that does the same as the others in ValueFederate.cpp now